spyder/plugins/editor/plugin.py

```diff
@@ -1074,7 +1082,8 @@ def get_plugin_actions(self):
         self.main.run_menu_actions = (
             run_menu_actions + self.main.run_menu_actions)
         run_toolbar_actions = [run_action, run_cell_action,
-                               run_cell_advance_action, run_selected_action]
+                               run_cell_advance_action, run_selected_action,
+                               create_new_cell]
```

Comment:
Let move this new action to the begining of the toolbar actions list for the moment, so you would get something like this: 

![image](https://user-images.githubusercontent.com/16781833/218288393-7c0edf1b-f7b7-4b2d-adf6-78bcae827579.png)